unstructured/ingest/connector/fsspec.py

```diff
@@ -90,6 +90,9 @@ class FsspecIngestDoc(IngestDocCleanupMixin, BaseIngestDoc):
     connector_config: SimpleFsspecConfig
     remote_file_path: str
 
+    def __post_init__(self):
+        self.update_source_metadata_metadata()
```

Comment:
this looks like a typo..in the original file assuming we should have called this `update_source_metadata` right?